public/locales/en/translation.json

```diff
@@ -878,6 +878,51 @@
         "addpath": "Add New Path",
         "removepath": "Remove Path"
     },
+    "tour": {
+        "back": "Back",
+        "done": "Done",
+        "library": {
+            "addGame": "Add your own games or apps to the library by clicking here. They can be basically anything, even Browser URLs.",
+            "categories": "Organize your games into categories to keep things tidy.",
+            "end": {
+                "intro": "That's it! Enjoy your games and have fun!",
+                "title": "Enjoy your games!"
+            },
+            "filters": "Filter your games by store, platform, favorites and more using these options.",
+            "gameCard": "Right Click on thegame card to navigate to the game page and to see details and adjust settings or Left click to open the context menu.",
+            "refresh": "Refresh your library to check for new games or updates.",
+            "search": "Use the Search bar to search for games by name or AppName.",
+            "sortInstalled": "Sort games by installed status.",
+            "sortOptions": "Click here to sort games alphabetically.",
+            "viewToggle": "Switch between grid and list view for your games.",
+            "welcome": {
+                "intro": "Welcome to the Heroic Library! This is where you can see all your games across different stores.",
+                "intro2": "If the library is empty, make sure to login with your accounts using the Manage accounts on the sidebar or add your own games using the Add Game button above.",
+                "title": "Welcome to Heroic!",
+                "title2": "Managing the library!"
+            }
+        },
+        "next": "Next",
+        "sidebar": {
+            "accessibility": "Access accessibility features to customize your experience.",
+            "accounts": "Manage your connected store accounts (Epic, GOG, Amazon), logout or login on them.",
```

Comment:
when a user opens heroic for the very first time, this `Manage accounts` button is not visible but the tour step is still going to it, pointing nowhere

<img width="978" alt="Screenshot 2025-03-09 at 7 02 43 PM" src="https://github.com/user-attachments/assets/e32ec21c-bec5-4ea4-8399-5bd70da086fb" />

we should have a tour step for the `Log in` button in this case

Reply:
added logic for that